### DISCUSSION TITLE
Add command to set Environment in Admin mode

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#658] [Ideas]
Could you please add admin command so that it can set environment to 'Administration' mode. Unless i am not looking in right place, i haven't seen this option anywhere.

It could be something like this:
`pac admin config --environment 'environment-url' --admin-mode true`
or
`pac admin config -env 'environment-url' -am true`

Thanks
Sanjeev

**Comment by davidjenni** [comment]
@sanjeev40084 thanks for the suggestion, we're adding this to our backlog. No ETA yet.

tracking internally as [AB#2682909](https://dev.azure.com/dynamicscrm/1fb98997-2f9e-4734-be8a-9728680447c2/_workitems/edit/2682909)

**Comment by sanjeev40084** [comment]
@petrochuk  i have posted this same request as discussion earlier. you can either close that and keep this or the other way. the other issue is #617

**Comment by sanjeev40084** [comment]
this is available now by running 'set-runtime-state' command. (available from [pac cli 1.30.6](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/1.30.6#releasenotes-body-tab))

`pac admin set-runtime-state -env 'env-id' -rs 'AdminMode'`
